### ISSUE TITLE
Exchange Rates "modifed at" does not update

### ISSUE THREAD

**Issue by primal100** [mirumee/django-prices-openexchangerates#28]
Hi,

If I understand correctly:

``` python manage.py update_exchange_rates --all```

Creates conversion rates in the database. However running the above command again results in exceptions that each conversion rate already exists. So to update the following is required:

```python manage.py update_exchange_rates```

Running this command does not update the modified_at column, so I think that needs to be changed, most likely in this line in tasks.py:

```conversion_rate.save(update_fields=['rate'])```

Also, I think the Saleor documentation will need to be updated to indicate how to update the rates, as currently it says:

"
To update the exchange rates run the following command at least once per day:

```python manage.py update_exchange_rates --all```
"

Whereas it should actually be run without -all.

http://saleor.readthedocs.io/en/latest/integrations/openexchangerates.html?highlight=update_exchange_rates

**Comment by rafalp** [comment]
`update_exchange_rates --all` seems like misnomer, considered that it calls `create_conversion_dates` internally. Perhaps this could be moved to `fetch_new_exchange_rates`? We would have clear flow then, with one admin command downloading new rates from API, and other command updating existing ones. Or we could make `update_exchange_rates` always sync our conversion rates with API, creating entries for missing currencies and updating existing ones.

/cc @patrys @elwoodxblues

**Comment by patrys** [comment]
@rafalp I'm not convinced. I've seen multiple update commands that take an explicit `--all` to mean "create new entities where necessary". Also "update" here means "make sure things are up-to-date", not nexcessarily "execute an SQL UPDATE command".

**Comment by patrys** [comment]
The bug is here:

https://github.com/mirumee/django-prices-openexchangerates/blob/b99cc11393825d2c4ece0aa9768fc6242269fab3/django_prices_openexchangerates/tasks.py#L52-L53

Instead the code should read something like:


```python
conversion_rate, created = ConversionRate.objects.get_or_create(
    to_currency=currency, defaults={'rate': rate})
if not created:
    conversion_rate.rate = rate
    conversion_rate.save(update_fields=['rate', 'modified_at'])
```

**Comment by rafalp** [comment]
Allright, I'll include the fix for this in the Beta 4 PR.

**Comment by rafalp** [comment]
Hey @primal100,

The original issue reported (`modifed_at` not updating) has been fixed in 1.0.0

I'll open separate issue for tracking bug that @patrys mentions above.
